doc/source/serve/production-guide/kubernetes.md

```diff
@@ -0,0 +1,184 @@
+`RayService` custom resources consist of the following:
+- a `KubeRay` `RayCluster` config defining the cluster that the Serve application runs on.
+- a Ray Serve [config](serve-in-production-config-file) defining the Serve application to run on the cluster.
+
+When the `RayService` is created, the `KubeRay` controller first creates a Ray cluster using the provided configuration.
+Then, once the cluster is running, it deploys the Serve application to the cluster using the [REST API](serve-in-production-deploying).
+The controller also creates a Kubernetes Service that can be used to route traffic to the Serve application.
+
+Let's see this in action by deploying the same [`FruitStand` example](serve-in-production-example) from above.
+The Serve config for the example is embedded into [this example `RayService` CR](https://github.com/ray-project/kuberay/blob/release-0.3/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml).
+To follow along, save this CR locally in a file named `ray_v1alpha1_rayservice.yaml`:
+
+```console
+$ curl -o ray_v1alpha1_rayservice.yaml https://raw.githubusercontent.com/ray-project/kuberay/release-0.3/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml
+```
+
+To deploy the example, we simply `kubectl apply` the CR.
+This creates the underlying Ray cluster, consisting of a head and worker node pod (see [Ray Clusters Key Concepts](../../cluster/key-concepts.rst) for more details on Ray clusters), as well as the service that can be used to query our application:
+
+```console
+$ kubectl apply -f ray_v1alpha1_rayservice.yaml
+
+$ kubectl get rayservices
+NAME                AGE
+rayservice-sample   7s
+
+$ kubectl get pods
+NAME                                                      READY   STATUS    RESTARTS   AGE
+rayservice-sample-raycluster-qd2vl-worker-small-group-bxpp6   1/1     Running   0          24m
+rayservice-sample-raycluster-qd2vl-head-45hj4             1/1     Running   0          24m
+
+$ kubectl get services
+NAME                                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                          AGE
+kubernetes                                         ClusterIP   10.100.0.1       <none>        443/TCP                                          62d
+# Services used internally by the KubeRay controller.
+rayservice-sample-head-svc                         ClusterIP   10.100.34.24     <none>        6379/TCP,8265/TCP,10001/TCP,8000/TCP,52365/TCP   24m
+rayservice-sample-raycluster-qd2vl-dashboard-svc   ClusterIP   10.100.109.177   <none>        52365/TCP                                        24m
+rayservice-sample-raycluster-qd2vl-head-svc        ClusterIP   10.100.180.221   <none>        6379/TCP,8265/TCP,10001/TCP,8000/TCP,52365/TCP   24m
+# The Serve service that we will use to send queries to the application.
```

Comment:
Consider separating this into its own code block and explain that we will want to use the name of this service in the next steps (otherwise it kind of blends in with the rest of the console output).

Reply:
added a note highlighting it